<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hasło główne</string>
  </property>
  <widget class="QPushButton" name="pushButton_save_master">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_manual1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_manual2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_GP">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_manual">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Podaj hasło manualnie</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_automatic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Wygeneruj hasło automatycznie</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Podaj hasło:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Powtórz hasło:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Wygenerowane hasło:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GP">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Generuj</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_letters">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Litery</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_digits">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cyfry</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_special_characters">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Znaki specjalne</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_master_title">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zmień swoje hasło główne</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Długość hasła:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_GP_length">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
